core/keyboard_nav/navigation.js

```diff
@@ -542,6 +542,9 @@ Blockly.Navigation.insertBlock = function(block, targetConnection) {
   if (bestConnection && bestConnection.isConnected() &&
       !bestConnection.targetBlock().isShadow()) {
     bestConnection.disconnect();
+  } else if (!bestConnection) {
+    Blockly.Navigation.warn(
+        'This block can not be inserted at the marked location. Moved to workspace.');
```

Comment:
Does this move the target connection to the workspace in this case or leave it where it is?

Reply:
It leaves the marked connection where it is.

Reply:
Update the warning to say it's being left where it is then.

Reply:
The block is moved to the workspace which is what I was trying to communicate with the "Moved to workspace". Would it make more sense for it to be "Block has been moved to the top level workspace?"

Reply:
In the copy paste case it is, but this method also gets called if you select a block in a stack and press 'i' to insert it. In that case, if the marked location is invalid does it stay in the stack or get moved to the workspace?

Reply:
ah yup forgot about that. I will take that part out since it doesn't apply to all scenarios.